dockers/docker-lldp/lldpmgrd

```diff
@@ -310,16 +304,16 @@ class LldpManager(daemon_base.DaemonBase):
 
                 (key, op, fvp) = sst_appdb.pop()
                 if (key != "PortInitDone") and (key != "PortConfigDone"):
-                    if fvp:
-                        fvp_dict = dict(fvp)
-
-                        # handle port status change
-                        if "oper_status" in fvp_dict:
-                            if "up" in fvp_dict.get("oper_status"):
-                                self.generate_pending_lldp_config_cmd_for_port(key)
-                            else:
-                                self.pending_cmds.pop(key, None)
-
+                    if op == "SET":
+                        if fvp and "up" in dict(fvp).get("oper_status",""):
+                            self.generate_pending_lldp_config_cmd_for_port(key, dict(fvp))
+                        else:
+                            self.pending_cmds.pop(key, None)
```

Comment:
if fvp is empty, should we do no-op?

Reply:
when we have SET command fvp should always be with values.

but I will change it in a way that:
self.pending_cmds.pop(key, None) 
will be executed only if oper_state is "down"